core/server/master/src/test/java/alluxio/master/block/BlockMasterTest.java

```diff
@@ -242,11 +255,86 @@ public void registerCleansUpOrphanedBlocks() throws Exception {
         RegisterWorkerPOptions.getDefaultInstance());
 
     // Check that the worker heartbeat tells the worker to remove the blocks.
-    alluxio.grpc.Command heartBeat = mBlockMaster.workerHeartbeat(workerId, null,
+    alluxio.grpc.Command heartBeat = mBlockMaster.workerHeartbeat(workerId, mCurrentClusterId, null,
         memUsage, NO_BLOCKS, NO_BLOCKS_ON_LOCATION, NO_LOST_STORAGE, mMetrics);
     assertEquals(orphanedBlocks, heartBeat.getDataList());
   }
 
+  @Test
+  public void TestNewWorkerRegisterWithMaster() throws IOException {
```

Comment:
Normally we don't need "test" in the test name. Also we use camel case here. So `newWorkerRegisterWithMaster()` will be a better name.

Reply:
done